README.md

```diff
@@ -7,55 +7,56 @@ These are guest names I gathered in the fall that I plan to reach out to. If you
 Copy the snippet below, add your details and submit a PR!
 
 ```
-| Full Name | @twitterHandle | Previous Career | ❌ | ❌ |
+| Full Name | [social-media-handle](url-to-social-media-profile)* | Previous Career | ❌ | ❌ |
```

Comment:
Please remove the asterisk here. I want this to remain as copy/paste-able as possible and that's just a small bit of unnecessary markup that I think people will forget to delete.

Reply:
You are right, I didn't think of that.